error.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="d-flex h-screen align-center justify-center">
+    <v-sheet class="text-center">

```

Comment:
Nice! One thing to take note of is that we always PascalCase for component names. So `VSheet` and not `v-sheet`. Even though both are valid syntax it is smart to keep to established conventions.